src/fvm_multicell.hpp

```diff
@@ -233,8 +247,8 @@ class fvm_multicell {
 //////////////////////////////// Implementation ////////////////////////////////
 ////////////////////////////////////////////////////////////////////////////////
 
```

Comment:
The `DivPolicy` is used in one location. I wonder if it needs to be its own template parameter? Maybe when we construct a cell we can pass the policy as a parameter to the constructor?